### PR TITLE
CPLT2-5469

### PR DESCRIPTION
**Pull request by cb-jeffduska**
Example custom values files for the new Helm installation process. 

cc: @cloudbees/team-cloud-platform 
cc: @atreadway 
cc: @knylander

### CODE REVIEW

**Comment by kerogers-cloudbees** [comment]
You have migration-value-examples.yaml and migration-values-examples.yaml, and they appear to be the same.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Welcome
+Welcome to the CloudBees examples repository. This repository was created as a place for storing. tutorials, examples, and other learning materials.
```
Comment (knylander) [inline README.md]:
```suggestion
Welcome to the CloudBees examples repository. This repository was created as a place for storing tutorials, examples, and other learning materials. 
```

**Comment by cb-jeffduska** [comment]
@kerogers-cloudbees I removed the duplicate file.

**Comment by cb-jeffduska** [comment]
Doh! Didn't mean to close this.

**Comment by cb-jeffduska** [comment]
Doh! Didn't mean to close this.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Welcome
+Welcome to the CloudBees examples repository. This repository was created as a place for storing tutorials, examples, and other learning materials. 
+
+## Examples Summary
```
Comment (knylander) [inline README.md]:
```suggestion
## Examples summary
```

File: helm-custom-value-file-examples/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# CloudBees Core Helm Installation Custom Value Example
```
Comment (atreadway) [inline helm-custom-value-file-examples/README.md]:
```suggestion
# CloudBees Core Helm installation custom value example
```

**Pull request merged**